Skeleton

Deman4ig opened the pull request.

Deman4ig committed:
add normalize.css and "Inter" font

Deman4ig committed:
Merge branch 'master' of github.com:js-tasks-ru/htmlcss-20210513_deman4ig

Deman4ig committed:
add solution for lesson-1 task-2 skeleton

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

GSemikozov commented on 04-lection2/02-skeleton/skeleton.css:
```diff
@@ -0,0 +1,50 @@
+    border-radius: 8px;
+}
+
+.skeleton__inner {
```
в данном случае это модификатор, поэтому - `.skeleton_inner`. Если хотим оставить элементом - тогда в html это должен быть отдельный html-элемент которые следует сразу за `<div class="skeleton">`. А вообще - можно проще: убрать модификатор / элемент, а паддинг перенести в `.skeleton`)

Deman4ig replied:
вариант с паддингом в .skeleton наверное больше всего нравится.

изначально пытался сделать mixin, но сейчас перечитал что __inner действительно должен быть вложен в skeleton...

GSemikozov commented on 04-lection2/02-skeleton/skeleton.css:
```diff
@@ -0,0 +1,50 @@
+.skeleton__line {
+    height: 1.25em;
+    margin-top: .5em;
+    background: linear-gradient(92.51deg, #DDE2E5 6.44%, #E2E6E8 38.47%, #F3F5F5 56.48%, #E3E7EA 77.5%, #DDE2E5 102.52%);;
```
в конце случайно `;` добавлено 2 раза

GSemikozov commented on 04-lection2/02-skeleton/skeleton.css:
```diff
@@ -0,0 +1,50 @@
+
+.skeleton__line {
+    height: 1.25em;
+    margin-top: .5em;
```
не у каждого line должен быть margin-top)

Deman4ig replied:
перенес в модификатор

Deman4ig committed:
moved padding from .skeleton__inner to .skeleton

removed double semicolon at line 37 in skeleton.css

moved margin-top from .skeleton__line to a separate modificator and
centered lines
